### PR TITLE
Update qiime info to show parallel config info

### PR DESCRIPTION
**Pull request by Oddant1**
Adds info about the current parallel config to `qiime info`

blocked by qiime2/qiime2#775

### CODE REVIEW

**Comment by ebolyen** [comment]
Looks like there's some merge conflicts now

**Comment by Oddant1** [comment]
@ebolyen the tests here aren't pulling the version of the framework with the necessary PR merged. They pass locally.

File: q2cli/builtin/info.py [hunk 1]
```diff
@@ -38,10 +38,20 @@ def _echo_plugins():
...
 @click.command(help='Display information about current deployment.',
                cls=ToolCommand)
-def info():
+@click.option('--config-level',
+              required=False,
+              default=1,
+              show_default=True,
+              type=click.IntRange(0, 3),
+              help='Specify the level of detail you want the config to be '
+                   'displayed in')
```
Comment (gregcaporaso) [inline q2cli/builtin/info.py]:
```suggestion
              help='The level of detail to be used for displaying the configuration summary.')
```

File: q2cli/builtin/info.py [hunk 2]
```diff
@@ -51,5 +61,31 @@ def info():
     click.secho('\nApplication config directory', fg='green')
     click.secho(q2cli.util.get_app_dir())
 
+    if config_level:
```
Comment (gregcaporaso) [inline q2cli/builtin/info.py]:
```suggestion
    if config_level > 0:
```

File: q2cli/builtin/info.py [hunk 3]
```diff
@@ -51,5 +61,31 @@ def info():
...
+            elif config_level == 3:
+                config['parsl'], _ = load_config_from_dict(config)
+
+            click.secho(f'\n{config}')
+
     click.secho('\nGetting help', fg='green')
     click.secho('To get help with QIIME 2, visit https://qiime2.org')
+
+    # When we have the final docs for this stuff up we should probably replace
+    # this with a reference to our docs which wil reference the parsl docs
```
Comment (gregcaporaso) [inline q2cli/builtin/info.py]:
```suggestion
    # this with a reference to our docs which will reference the parsl docs
```

Comment (gregcaporaso) [reply q2cli/builtin/info.py]: Agreed.

**Review by gregcaporaso — CHANGES_REQUESTED** [review]
Just some minor suggestions on this. I pulled it down and tested locally, and it all looks right to me.

File: q2cli/builtin/info.py (hunk 3 above)
Comment (gregcaporaso) [reply q2cli/builtin/info.py]: We could alternatively reference this for now, and get the relevant references to the Parsl docs in there:
https://develop.qiime2.org/en/latest/framework/how-to-guides/parallel-configuration.html

That URL will probably change at some point though, which is my only concern.

Comment (Oddant1) [reply q2cli/builtin/info.py]: Yeah which could cause old versions of QIIME 2 to produce a URL that 404s. Those docs already link to the parsl docs in multiple places, but the first link there 404s...

Comment (Oddant1) [reply q2cli/builtin/info.py]: @ebolyen what's your opinion?

Comment (gregcaporaso) [reply q2cli/builtin/info.py]: @Oddant1, let's discuss strategy for this in the engineering meeting today.

**Comment by Oddant1** [comment]
@gregcaporaso I have updated the link to https://develop.qiime2.org/en/latest/framework/how-to-guides/parallel-configuration.html for now and opened an issue saying to change it when the docs are actually out.

**Review by gregcaporaso — APPROVED** [review]

**Comment by gregcaporaso** [comment]
Thanks again for the work on this one @Oddant!

**Pull request merged**